src/Transaction.purs

```diff
@@ -44,29 +48,60 @@ derive instance Eq ModifyTxError
 instance Show ModifyTxError where
   show = genericShow
 
--- | Set the `Transaction` body's script data hash. NOTE: Must include all of
+-- | Sets the script integrity hash and attaches redeemers, for use after
+-- | reindexing
+finalizeTransaction
```

Comment:
Should we have an `UnattachedReindexedTx` or something for this? `UnbalancedUnattachedTx` doesn't work because the transaction might be balanced

Reply:
Good idea